[Merged by Bors] - Add demo kafka-druid-earthquake-data (#72)
stackabletech/stackablectl: docs/modules/ROOT/pages/quickstart.adoc

== Browse available demos
Stackable provides a set of ready-to-use demos.
As of writing (2022/08/15) only two demos are available, but further demos will be added in the future.
They will automatically show up, as `stackablectl` fetches the available list of demos via the internet.
To list the available demos run the following command:

[source,console]
----
$ stackablectl demo list
DEMO                                STACKABLE STACK           DESCRIPTION
trino-taxi-data                     trino-superset-s3         Demo loading 2.5 years of New York taxi data into S3 bucket, creating a Trino table and a Superset dashboard

> kafka-druid-earthquake-data         kafka-druid-superset-s3   Demo ingestion earthquake data into Kafka, streaming it into Druid and creating a Superset dashboard

Reviewer: ```suggestion
kafka-druid-earthquake-data         kafka-druid-superset-s3   Demo to ingest earthquake data into Kafka, streaming it into Druid and creating a Superset dashboard
```

[INFO ] Installed demo trino-taxi-data. Use "stackablectl services list" to list the installed services
----

> The demos create Kubernetes jobs, that will populate testdata and talk to the installed products to process the data.

Reviewer: ```suggestion
The demos create Kubernetes jobs, that will populate test data and talk to the installed products to process the data.
```

Until the products are ready, it is completely normal that the pods of these Jobs will fail with an error.
They will get retried with an exponentially growing backoff time.

> After the products being ready they should turn green and everything should settle down.

Reviewer: ```suggestion
After the products are ready they should turn green and everything should settle down.
```
